### PR TITLE
TS-4211: Make freelist_new cleaner so we can debug better.

### PR DESCRIPTION
**Pull request by PSUdaemon**

### CODE REVIEW

File: lib/ts/ink_queue.cc [hunk 1]
```diff
@@ -200,19 +200,20 @@ freelist_new(InkFreeList *f)
     if (TO_PTR(FREELIST_POINTER(item)) == NULL) {
       uint32_t i;
       void *newp = NULL;
-      size_t alloc_size = 0;
+      size_t alloc_size = f->chunk_size * f->type_size;
+      size_t alignment;
```
Comment (bgaff) [inline lib/ts/ink_queue.cc]: = 0.

**Comment by bgaff** [comment]
All good :+1: just set alignment = 0 on declaration and let the complier deal with it if it's always assigned later.

**Comment by bgaff** [comment]
I just confirmed that this is correct, it will actually be hugepage aligned.

**Comment by PSUdaemon** [comment]
I addressed the comments above. At some point in the future I want to replace this with a page allocator for the freelists but that is going to be a bigger task.

**Comment by PSUdaemon** [comment]
Merged.

**Pull request closed without merging**